flexible-classroom/overview/product-overview.mdx

```diff
@@ -9,10 +9,9 @@ description: >
 <ProductOverview
     title="Flexible classroom"
     img="/images/product-overview/video-calling.png"
-    quickStartLink="/flexible-classroom/get-started/get-started-sdk"
-    uiQuickStartLink="/flexible-classroom/get-started/get-started-uikit"
+    quickStartLink="/flexible-classroom/get-started/get-started"
     apiReferenceLink="/api-reference"
-    samplesLink="#"
+    samplesLink="/flexible-classroom/reference/downloads"
```

Comment:
Suggestion: You can replace the sample links with this: https://docs.agora.io/en/agora-class/code-samples?platform=All%20Platforms.
Reason: The current so-called sample links in this PR are demo apps, not GitHub sample project links. 

**Another important thing** I want to share with you is that, Flexible Classroom a aPaaS product, not a PaaS product. So we do not provide SDKs for users, instead, we provide demo apps (you can find them on this page: https://docs.agora.io/en/agora-class/downloads?platform=All%20Platforms). For the web platform, we provide both Web and H5 demo apps; for the Electron platform, we provide both Electron macOS and Electron Windows demo apps. So, I think besides adding the GitHub sample links here, we need to add the demo app links to the SDK page. If you think it's weird to add demo app links to the SDK page, you can add another line after this line, saying `demosLink="xxx"`, and add a reference doc to write the description and link for demo apps.

Reply:
How weird is that, now have both.